frontend/src/types/help.ts

```diff
@@ -0,0 +1 @@
+export type routeMap = Record<"routeName" | "helpName", string>[];
```

Comment:
Types should be named using the "CamelCase" convention. And this type should be the object structure of the helper data.